src/core/lwt.ml

```diff
@@ -1,9 +1,9 @@
-(* Lightweight thread library for OCaml
+(* OCaml promise library
```

Comment:
If you want to play code archeology, I would suggest looking for jérôme's code inside the [unison](https://www.cis.upenn.edu/~bcpierce/unison/) project.